x-pack/solutions/observability/plugins/observability/public/pages/alert_details/components/state_alert.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useMemo, useState } from 'react';
+import type { Rule } from '@kbn/triggers-actions-ui-plugin/public';
+import { ALERT_CASE_IDS, ALERT_STATUS_ACTIVE, ALERT_UUID } from '@kbn/rule-data-utils';
+import moment from 'moment';
+import { EuiButton, EuiCallOut, EuiFlexGroup } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { RuleFormFlyout } from '@kbn/response-ops-rule-form/flyout';
+import { useKibana } from '../../../utils/kibana_react';
+import { AlertData } from '../../../hooks/use_fetch_alert_detail';
+import { useBulkUntrackAlerts } from '../hooks/use_bulk_untrack_alerts';
+
+function StaleAlert({
+  alert,
+  alertStatus,
+  rule,
+  refetchRule,
+  onUntrackAlert,
+}: {
+  alert: AlertData;
+  alertStatus: string | undefined;
+  rule: Rule | undefined;
+  refetchRule: () => void;
+  onUntrackAlert: () => void;
+}) {
+  const { services } = useKibana();
+  const {
+    triggersActionsUi: { ruleTypeRegistry, actionTypeRegistry },
+  } = services;
+  const [ruleConditionsFlyoutOpen, setRuleConditionsFlyoutOpen] = useState<boolean>(false);
+  const { mutateAsync: untrackAlerts } = useBulkUntrackAlerts();
+  const handleUntrackAlert = useCallback(async () => {
+    const alertUuid = alert?.formatted.fields[ALERT_UUID];
+    if (alertUuid) {
+      await untrackAlerts({
+        indices: ['.internal.alerts-observability.*'],
+        alertUuids: [alertUuid],
+      });
+      onUntrackAlert();
+    }
+  }, [alert?.formatted.fields, untrackAlerts, onUntrackAlert]);
+  const handleEditRuleDetails = () => {
+    setRuleConditionsFlyoutOpen(true);
+  };
+  const isAlertStale = useMemo(() => {
+    if (alertStatus === ALERT_STATUS_ACTIVE) {
+      const numOfCases = alert?.formatted.fields[ALERT_CASE_IDS]?.length || 0;
```

Comment:
can `alert` be undefined? 🤔

Reply:
Yes, it's fetched from the alert_details page file via `const [isLoading, alertDetail] = useFetchAlertDetail(alertId);`

Reply:
we are already checking [here](https://github.com/elastic/kibana/blob/98efc8be21ecfac89b5a5da81388456339d62426/x-pack/solutions/observability/plugins/observability/public/pages/alert_details/alert_details.tsx#L222) if alertDetail is undefined